quizcon/templates/main/lti_assignment.html

```diff
@@ -5,24 +5,62 @@
 
 {% load coursetags static %}
 {% block topnavbar %}
-{% if is_faculty %}
-    <nav class="navbar navbar-expand-lg navbar-dark">
-        <div class="container-fluid">
-            <a class="navbar-brand" href="{{ request.scheme }}://{{ request.get_host }}{% url 'course-detail-view' quiz.course.pk %}" target="_blank" rel="noopener noreferrer"><strong><span class="text-light">Quizzing With Confidence</span></strong></a>
-            <div class="collapse navbar-collapse" id="nabarNav">
-                <ul class="navbar-nav">
-                    <li class="nav-item"> <a class="nav-link" href="{% url 'update-quiz' quiz.id %}" target="_blank" rel="noopener noreferrer">Edit Quiz</a></li>
-                </ul>
+    {% if is_faculty %}
+        <nav class="navbar navbar-expand-lg navbar-dark">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="{{ request.scheme }}://{{ request.get_host }}{% url 'course-detail-view' quiz.course.pk %}" target="_blank" rel="noopener noreferrer"><strong><span class="text-light">Quizzing With Confidence</span></strong></a>
+                <div class="collapse navbar-collapse" id="nabarNav">
+                    <ul class="navbar-nav">
+                        <li class="nav-item"> <a class="nav-link" href="{% url 'update-quiz' quiz.id %}" target="_blank" rel="noopener noreferrer">Edit Quiz</a></li>
+                    </ul>
+                </div>
             </div>
-        </div>
-    </nav>
-{% else %}
-    <nav class="navbar navbar-expand-lg navbar-dark">
-        <div class="container-fluid">
-            <a class="navbar-brand" href="#"><strong><span class="text-light">Quizzing With Confidence</span></strong></a>
-        </div>
-    </nav>
-{% endif %}
+        </nav>
+    {% else %}
+        <nav class="navbar navbar-expand-lg navbar-dark">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="#"><strong><span class="text-light">Quizzing With Confidence</span></strong></a>
+            </div>
+        </nav>
+    {% endif %}
+{% endblock %}
+
+{% block js %}
+    <script>
+        var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'))
```

Comment:
The tooltips look great. Is there anyway to remove the "hover" functionality? It's pretty distracting to have both hover & click functionality.

Reply:
I made it so that the trigger is only a click. As of now the tooltip doesn't go away when another button is clicked. I was trying to find a solution but I am wondering if it is helpful for the student to be able to compare all points at the same time.

Reply:
I don't mind them staying open. The only issue is that when you navigate to the next page, they are still open. :-D

Here's some example code on closing open menus when a user "clicks out" of the menu. This could be used to close the tooltip after each click. Otherwise, you'll need to add some code to close them on Prev & Next.
https://github.com/ccnmtl/mediathread/blob/master/mediathread/templates/base.html#L334